src/gui/tray/UserModel.h

```diff
@@ -132,6 +139,8 @@ class UserModel : public QAbstractListModel
     Q_INVOKABLE bool currentUserHasLocalFolder();
     int currentUserId() const;
     Q_INVOKABLE bool isUserConnected(const int &id);
+    Q_INVOKABLE bool isNotificationStatusOnline(const int &id);
```

Comment:
I'd avoid the Q_INVOKABLEs and have properties instead. @nicolasfella started cleaning that up, let's not introduce more.

And to receive int, no need for const ref: `void foo(int id);` does the job best for those types.

Reply:
In the end I was not even using this Q_INVOKABLE. Removed it.